multiversx_sdk_cli/cli.py

```diff
@@ -53,6 +55,11 @@ def _do_main(cli_args: List[str]):
         args.func(args)
 
 
+def ensure_multiversx_sdk_directory_exists() -> None:
+    sdk_path = Path("~/multiversx-sdk").expanduser().resolve()
```

Comment:
I think we have this as a constant in `workstation.py` or something similar.

Reply:
Removed `ensure_multiversx_sdk_directory_exists()`. Now using `utils.ensure_folder(config.SDK_PATH)`.
Thanks for the suggestion.